### PR TITLE
Use asserts to verify that our assumptions are held

### PR DESCRIPTION
**Pull request by magicus**

### CODE REVIEW

**Comment by kristofbolyai** [comment]
Another good PR for settings screen 😀

**Comment by magicus** [comment]
@kristofbolyai I'm doing what I can to assist you. The settings screen is necessary, and a lot of work. And it inspires me to raise the quality of other, settings-related stuff in Artemis.

File: common/src/main/java/com/wynntils/core/keybinds/KeyBindManager.java [hunk 1]
```diff
@@ -42,7 +42,7 @@ public static void onKeyPress(InventoryKeyPressEvent e) {
 
     public static void registerKeybind(KeyBind toAdd) {
         if (hasName(toAdd.getName())) {
-            throw new IllegalStateException("Can not add " + toAdd + " since the name already exists");
+            throw new IllegalStateException("Can not add kebind " + toAdd.getName() + " since the name already exists");
```
Comment (HighCrit) [inline common/src/main/java/com/wynntils/core/keybinds/KeyBindManager.java]: “Kebind”

**Review by HighCrit — CHANGES_REQUESTED** [review]

File: common/src/main/java/com/wynntils/core/keybinds/KeyBindManager.java (hunk 1 above)
Comment (magicus) [reply common/src/main/java/com/wynntils/core/keybinds/KeyBindManager.java]: Kaboom.

**Review by HighCrit — APPROVED** [review]

**Review by kristofbolyai — APPROVED** [review]

**Comment by kristofbolyai** [comment]
We should make a PR to enable assertions in our Github Action build.

**Comment by magicus** [comment]
Should we? We're not running it, so it won't show anything. That will only show like assertions in Gradle...

**Comment by kristofbolyai** [comment]
Ah right. No brain moment. I just wanted a way to test PRs.

**Comment by magicus** [comment]
With that being said, it would be fantastic if we had a way to run tests.

But the only ways I can think of require tons of work, and would probably not be able to make legally on GHA. :(

**Pull request merged**